### DISCUSSION TITLE
Deleted or private repositories appear on feed

### DISCUSSION THREAD

**Discussion by llrs** [community/community#13263] [Other Feature Feedback, Questions, & Ideas]
I have found that there are notifications of repositories that are either deleted or private but they show up on my following feed (I click on them but I hit a 404). Perhaps is due to a cache as they might have been previously public, or there might create too many queries to the database and this is not frequently refreshed.   
It would be nice if these feeds were we can't do anything with it could be reduced. 

Not sure if this wasn't the case before (just noticed this today)

**Comment by bdresser** [comment]
Thanks for the feedback @llrs. Can you share an example of where you saw this happen? If you'd rather not post a screenshot here, feel free to email me at bdresser at github dot com.

**Reply by llrs** [reply]
Here it is:

![Screenshot from 2022-03-22 18-21-28](https://user-images.githubusercontent.com/6818218/159539005-d1ee9c16-ffb2-4060-8606-d2e105136fb8.png)

**Reply by llrs** [reply]
That user was testing/learning git+hub so maybe the repo was created and deleted several times (but AFAIK the repo exists but is private)

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by llrs** [reply]
Now I cannnot find any private repository on the feed. But the filter menu doesn't provide an option for those "updates". I think I am missing something

**Reply by jfullstackdev** [reply]
maybe already fixed by them ?

**Reply by llrs** [reply]
I don't know and I cannot confirm it.
